components/02-molecules/text-with-image/text-with-image.twig

```diff
@@ -0,0 +1,72 @@
+{#
+ # Available Props:
+ # - text_with_image__width: feature (default) or highlight
+ # - text_with_image__position: image-left (default) or image-right
+ # - text_with_image__focus: equal (default), image, or text
+ #
+ # Available variables:
+ # - text_with_image__overline (optional)
+ # - text_with_image__heading
+ # - text_with_image__subheading (optional)
+ # - text_with_image__text
+ # - text_with_image__link__content (optional)
+ # - text_with_image__link__url (optional)
+ #}
+
+{% set text_with_image__base_class = 'text-with-image' %}
+
+{% set text_with_image__attributes = {
+  'data-component-focus': text_with_image__focus|default('equal'),
+  'data-image-position': text_with_image__position|default('image-left'),
+  'data-component-width': text_with_image__width|default('feature'),
+  class: bem(text_with_image__base_class)
+} %}
+
+<div {{ add_attributes(text_with_image__attributes) }}>
+  <div {{ bem('inner', [], text_with_image__base_class) }}>
+    {# Content #}
+    <div {{ bem('content', [], text_with_image__base_class) }}>
+      {# Overline #}
+      {% if text_with_image__overline %}
+        {% include "@atoms/typography/text/text.twig" with {
+          text__content: text_with_image__overline,
+          text__base_class: 'overline',
+          text__blockname: text_with_image__base_class,
+        } %}
+      {% endif %}
+      {# Heading #}
+      {% include "@atoms/typography/headings/heading.twig" with {
+        heading: text_with_image__heading,
+        heading__level: '2',
+        heading__blockname: text_with_image__base_class,
+      } %}
+      {# Subheading #}
+      {% if text_with_image__subheading %}
+        {% include "@atoms/typography/headings/heading.twig" with {
+          heading: text_with_image__subheading,
+          heading__level: '3',
```

Comment:
Looking really good!! 

The subheading doesn't need to be semantically a heading, it's not a new section or subsection of the parent heading. 

For a screen reader user, having the dates as H3's will clutter their web rotor (a tool that they use pull out headings/landmarks/links in the DOM, shown below). 

<img width="847" alt="Screen Shot 2022-03-11 at 9 58 19 AM" src="https://user-images.githubusercontent.com/12991593/157893682-e0a1ed53-3a61-4ed3-b61f-df6c4b3f1f8a.png">